includes/payment-tokens/class-wc-payment-token-cc.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+/**
+ * WooCommerce Credit Card Payment Token
+ *
+ * Representation of a payment token for credit cards
+ * Extends from WC_Payment_Token_eCheck since both types contain last4 digits.
+ *
+ * @class 		WC_Payment_Token_CC
+ * @since		2.6.0
+ * @category 	PaymentTokens
+ * @package 	WooCommerce/PaymentTokens
+ * @author		WooThemes
+ */
+class WC_Payment_Token_CC extends WC_Payment_Token_eCheck {
```

Comment:
I think #1 is okay - since they both have a common field/validation which is why I think it makes sense to do. However I agree #2 could be a problem in the future. @mikejolley what do you think here?